kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopEventManager.java

```diff
@@ -242,18 +251,26 @@ private JsonObject parseJsonObject(String info) {
     }
 
     @VisibleForTesting
-    public static Node getNode(String kopBrokerStr) {
-        final String errorMessage = "kopBrokerStr " + kopBrokerStr + " is invalid";
-        final Matcher matcher = PATTERN.matcher(kopBrokerStr);
-        checkState(matcher.find(), errorMessage);
-        checkState(matcher.groupCount() == 3, errorMessage);
-        String host = matcher.group(2);
-        String port = matcher.group(3);
-
-        return new Node(
-                Murmur3_32Hash.getInstance().makeHash((host + port).getBytes(StandardCharsets.UTF_8)),
-                host,
-                Integer.parseInt(port));
+    public static Map<String, Set<Node>> getNodes(String kopBrokerStrs) {
+        HashMap<String, Set<Node>> nodesMap = Maps.newHashMap();
+        String[] kopBrokerArr = kopBrokerStrs.split(END_POINT_SEPARATOR);
+        for (String kopBrokerStr : kopBrokerArr) {
+            final String errorMessage = "kopBrokerStr " + kopBrokerStr + " is invalid";
+            final Matcher matcher = PATTERN.matcher(kopBrokerStr);
+            checkState(matcher.find(), errorMessage);
+            checkState(matcher.groupCount() == 3, errorMessage);
+            String listenerName = matcher.group(1);
+            String host = matcher.group(2);
+            String port = matcher.group(3);
```

Comment:
This parse is nearly the same with what `EndPoint` does. Could you reuse the existing code? For example, add a conversion from `EndPoint` to `Node` or add some utils methods?